bower.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "vui-button",
-  "main": "button.scss",
+  "main": "button.html",
   "license": "Apache-2.0",
   "ignore": [
-    "node_modules",
-    "bower_components",
     "test",
-    ".csslintrc",
     ".editorconfig",
     ".gitignore",
-    ".npmignore",
     ".travis.yml",
     "package.json"
   ],
   "dependencies": {
-    "vui-colors": "^0.2.0",
-    "vui-typography": "^2.0.0"
```

Comment:
Yeah, I was thinking along those lines too when I was looking at the small-text mixins in vui-typography.